Sources/Slacket/Services/BackendService.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  BackendService.swift
+//  Slacket
+//
+//  Created by Jakub Tomanik on 20/05/16.
+//
+//
+
+import Foundation
+
+protocol BackendService {
```

Comment:
Base protocol for all services. I can imagine we may want to have some common properties for all services ie. `path`. So for now this is only a placeholder.